packages/cnpj-val/tests/Specs/Exceptions.spec.php

```
@@ -0,0 +1,245 @@
+        it('has the correct name', function () {
+            $error = new CnpjValidatorInputTypeError(123, 'string or string[]');
+
+            expect($error->getName())->toBe('CnpjValidatorInputTypeError');
+        });
+    });
+});
+
+describe('CnpjValidatorOptionsTypeError', function () {
+    describe('when instantiated', function () {
+        it('is an instance of `TypeError`', function () {
+            $error = new CnpjValidatorOptionsTypeError('hidden', 123, 'boolean');
```

PHPStan (2.1.54)

[warning] Parameter `#1` $optionName of class Lacus\BrUtils\Cnpj\Exceptions\CnpjValidatorOptionsTypeError constructor expects 'caseSensitive'|'type', 'hidden' given.


(argument.type)